### DISCUSSION TITLE
Does llama.cpp have support for Llama-3.2 yet?

### DISCUSSION THREAD

**Discussion by pseelam02** [ggml-org/llama.cpp#9731] [Q&A]
I have been trying to use the following model https://huggingface.co/unsloth/Llama-3.2-3B-Instruct-GGUF with the llama.cpp pipeline. However, when trying to run the model with llama.cpp I get the following error: 
![image](https://github.com/user-attachments/assets/879150c3-82ab-4f7c-84c2-74b5c283038b)

Any help would be greatly appreciated!
<img width="718" alt="Screenshot 2024-10-03 at 11 02 16 AM" src="https://github.com/user-attachments/assets/b202e0e7-07d1-46b2-a056-2498d0117884">

**Comment by supportend** [comment]
"task = transcribe", are you try this with whisper.cpp?
Llama 3.2 works with recent llama.cpp.

**Reply by pseelam02** [reply]
yes, I am trying it with with whisper.cpp
